db/migrate/20180710172959_disallow_multi_levels_theme_components.rb

```diff
@@ -0,0 +1,66 @@
+def already_exists?(parent, child)
+  DB.query("SELECT 1 AS one FROM child_themes WHERE child_theme_id = :child AND parent_theme_id = :parent", child: child, parent: parent).present?
+end
+
+class DisallowMultiLevelsThemeComponents < ActiveRecord::Migration[5.2]
+  def up
+    handeled = []
+    proc = Proc.new do |parent, top_parent|
```

Comment:
Is there a reason we use a proc here instead of defining a method? Seems abit odd to me.